### PR TITLE
better error handling

### PR DESCRIPTION
**Pull request by antoine-de**
less logs and better handling of the stop_area and visible flags

### CODE REVIEW

File: src/bin/stops2mimir.rs [hunk 1]
```diff
@@ -112,10 +112,12 @@ impl<'a, R: std::io::Read + 'a> Iterator for StopPointIter<'a, R> {
         }
 
         fn is_valid_stop_area(location_type: &Option<u8>, visible: &Option<u8>) -> csv::Result<()> {
-            if *location_type == Some(1) && *visible != Some(0) {
-                Ok(())
+            if *location_type == Some(0) {
+                Err(csv::Error::Decode("not a stop_area.".to_string()))
```
Comment (TeXitoi) [inline src/bin/stops2mimir.rs]: No, we want it equal to 1 to be a stop area.

File: tests/fixtures/stops.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 stop_id,visible,stop_name,stop_lat,stop_lon,zone_id,location_type,parent_station,stop_timezone,equipment_id,contributor_id,geometry_id,frame_id
 SA:main_station,1,"République",47.099758,2.491858,,1,,Europe/Paris,,BGT,,BGT:19
-SA:second_station,,"14 Juillet",47.123966,2.402522,,1,,Europe/Paris,,BGT,,BGT:19
+SA:second_station,,"14 Juillet",47.123966,2.402522,,,,Europe/Paris,,BGT,,BGT:19
```
Comment (TeXitoi) [inline tests/fixtures/stops.txt]: That's not what you want.

File: src/bin/stops2mimir.rs (hunk 1 above)
Comment (antoine-de) [inline src/bin/stops2mimir.rs]: since it's an optional field in GTFS we'll thus only have stop_points when we want to have stop areas no ?

Comment (antoine-de) [inline src/bin/stops2mimir.rs]: ah yes sorry misread the doc, in this case we want all to be stoppoints, I'll change this

**Pull request merged**